emva1288/process/plotting.py

```diff
@@ -370,9 +370,9 @@ def plot(self, test):
 class PlotHorizontalSpectrogramPRNU(Emva1288Plot):
     '''Create Horizontal spectrogram PRNU plot'''
 
-    name = 'Horizontal spectrogram PRNU'
-    xlabel = 'cycles [periods/pixel]'
-    ylabel = 'Standard deviation and\nrelative presence of each cycle [%]'
+    name = r'Horizontal spectrogram PRNU'
+    xlabel = r'cycles [periods/pixel]'
+    ylabel = r'Standard deviation and\nrelative presence of each cycle [%]'
```

Comment:
We want `\n` to actually be treated as a newline character here.
```suggestion
    ylabel = 'Standard deviation and\nrelative presence of each cycle [%]'
```

Reply:
As you can see, the newline is not treated correctly with your proposed change.
![image](https://user-images.githubusercontent.com/862398/191961025-ca52e177-9d28-4809-a319-25150b1bc5df.png)